packages/calling-component-bindings/src/videoGallerySelector.ts

```diff
@@ -89,12 +97,19 @@ export const videoGallerySelector: VideoGallerySelector = createSelector(
     const dominantSpeakersMap: Record<string, number> = {};
     dominantSpeakerIds?.forEach((speaker, idx) => (dominantSpeakersMap[speaker] = idx));
     const noRemoteParticipants = [];
+
+    let convert = convertRemoteParticipantToVideoGalleryRemoteParticipant as any;
```

Comment:
If we are doing this in the selector utils do we need to do this here too?

Reply:
thanks, rewrote it